src/server.ts

```diff
@@ -335,7 +336,13 @@ export class BlobsServer {
       return {}
     }
 
-    const [, siteID, rawStoreName, ...key] = url.pathname.split('/')
+    let parts = url.pathname.split('/').slice(1)
+
+    if (parts[0].startsWith(REGION_PREFIX)) {
+      parts = parts.slice(1)
```

Comment:
we're discarding the `region` parameter, which means that in local dev, stores with different `experimentalRegion` parameters all share the same data. Is that exepcted?

Reply:
I think that's fine for now. We will likely need to rethink the whole namespacing we use locally once we make region selection a customer-facing thing, but I think just ignoring regions for now is an okay first step.